### DISCUSSION TITLE
Cadence examples are timed out

### DISCUSSION THREAD

**Discussion by MrSyee** [uber/cadence#5057] [Q&A]
I tried to setup Cadence server followed [this tutorial](https://cadenceworkflow.io/docs/get-started/installation/).
And I registered a domain using Cadence CLI (I installed CLI using `brew install`).
I'd like to execute workflow so I clone [cadence-samples](https://github.com/uber-common/cadence-samples) and then tried to build and execute `helloworld` example.
But this example is timed out error. What should I do?
The others examples are also timed out.
![image](https://user-images.githubusercontent.com/17582508/209070831-a584b8c6-93e4-4f61-9aa2-8fbf44f60a16.png)
![image](https://user-images.githubusercontent.com/17582508/209071013-54b0c591-08dd-4dfe-bf31-36770c5ee9c4.png)

**Comment by Mypmahck** [comment]
您发给我的信件已经收到。
